test/WebJobs.Script.Tests/DependencyTests.cs

```diff
@@ -18,6 +18,8 @@ public class DependencyTests
         // These are changed often and controlled by us, so we don't need to fail if they are updated.
         private static readonly string[] _excludedList = new[]
         {
+            "Microsoft.Azure.WebJobs.Script.dll",
```

Comment:
@brettsam how were these not already on the list ? :)

Reply:
hmm, dunno -- they have never triggered a failure for some reason?